tests/suites/test_suite_ssl.function

```diff
@@ -3806,6 +3806,71 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE depends_on:MBEDTLS_TEST_HOOKS:MBEDTLS_SSL_PROTO_TLS1_3 */
+void psa_hkdf_extract( int alg, char *hex_ikm_string,
```

Comment:
This is partially pre-existing, as these test are copied from `test_suite_hkdf.function`, so I was on the fence about asking for that. If we are using `data_t` here, it would probably make sense to change the tests in `test_suite_hkdf` as well, and I guess it wouldn't be much more work if you do it at the same time.

Reply:
A very long time ago, we didn't have `data_t` and each test function had to convert inputs from hex to bin. Since then we've added `data_t` and reworked a lot of tests to use it, but there are a few remaining ones that take `char *` hex inputs. Please don't do that in new code.